### PR TITLE
Updating the RELNOTES for 1.3.0rc0.

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,99 @@
...
+Carbone, Kevin Van Der Burgt, Kongsea, ksellesk, lanhin, Lef Ioannidis, Liangliang He,
+Louis Tiao, Luke Iwanski, LáSzló Csomor, magixsno, Mahmoud Abuzaina, Marcel Hlopko, Mark
+Neumann, Maxwell Paul Brickner, mdfaijul, MichaëL Defferrard, Michał JastrzęBski, Michele
+Colombo, Mike Brodie, Mosnoi Ion, mouradmourafiq, myPrecious, Nayana Thorat,
+Neeraj Kashyap, Nelson Liu, Niranjan Hasabnis, Olivier Moindrot, orome, Pankaj Gupta, Paul
+Van Eck, peeyush18, Peng Yu, Pierre, preciousdp11, qjivy, Raingo, raoqiyu, ribx, Richard S.
+Imaoka, Rishabh Patel, Robert Walecki, Rockford Wei, Ryan Kung, Sahil Dua, Sandip Giri, Sayed
+Hadi Hashemi, sgt101, Shitian Ni, Shuolongbj, Siim PõDer, Simon Perkins, sj6077, SOLARIS,
+Spotlight0xff, Steffen Eberbach, Stephen Fox, superryanguo, Sven Mayer, Tapan Prakash,
+Tiago Morais Morgado, Till Hoffmann, Tj Rana, Todd Wang, Vadim Markovtsev,
```
Comment (drpngx) [inline RELEASE.md]: Todd Wang is at google.

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
+* `tf.RewriterConfig` was removed from the Python API after being available in 1.2 release candidates (it was never in an actual release). Graph rewriting is still available, just not as `tf.RewriterConfig`. Instead add an explicit import.
+* Breaking change to `tf.contrib.data.Dataset` APIs that expect a nested structure. Lists are now converted to `tf.Tensor` implicitly. You may need to change uses of lists to tuples in existing code. In addition, dicts are now supported as a nested structure.
+
+## Changes to contrib APIs
+* Adds tf.contrib.nn.rank_sampled_softmax_loss, a sampled-softmax variant that can improve rank loss.
+* `tf.contrib.metrics`.{streaming_covariance,streaming_pearson_correlation} modified to return nan when they have seen less or equal to 1 unit of weight.
+* Adds time series models to contrib. See contrib/timeseries/README.md for details.
+* Adds FULLY_CONNECTED Op to tensorflow/contrib/lite/schema.fbs
+
+## Bug Fixes and Other Changes
```
Comment (caisq) [inline RELEASE.md]: Add
* TensorFlow Debugger (tfdbg):
** Fixed a bug that prevent tfdbg from functioning with multi-GPU setups.
** Fixed a bug that prevent tfdbg from working with `tf.Session.make_callable`.

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,99 @@
+# Release 1.3.0
+
+## Major Features and Improvements
+* Added canned estimators to Tensorflow library. List of added estimators: DNNClassifier, DNNRegressor, LinearClassifer, LinearRegressor, DNNLinearCombinedClassifier, DNNLinearCombinedRegressor.
```
Comment (drpngx) [inline RELEASE.md]: Probably quote `DNNClassifier` etc

**Review by drpngx — APPROVED** [review]

File: RELEASE.md [hunk 4]
```diff
@@ -1,3 +1,99 @@
...
+* Add `ConcatenateDataset` to concatenate two datasets.
+* Added Mobilenet support to TensorFlow for Poets training script.
+* Adds a block cache to the GCS filesystem with configurable block size and count.
+* SinhArcSinh bijector added.
+* Added `Dataset.list_files` API.
+* Introduces new operations and Python bindings for the Cloud TPU.
+* Adding TensorFlow-iOS CocoaPod for symmetry with tensorflow-android.
+* Introduces base implementations of ClusterResolvers.
+* Unify memory representations of TensorShape and PartialTensorShape. As a consequence, tensors now have a maximum of 254 dimensions, not 255.
+* Changed references to LIBXSMM to use version 1.8.1.
```
Comment (caisq) [inline RELEASE.md]: Add:
* TensorFlow Debugger (tfdbg): Display summaries of numeric tensor values with the `-s` flag to command `print_tensor` or `pt`.

**Review by caisq — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 1 above)
Comment (av8ramit) [inline RELEASE.md]: Done.

File: RELEASE.md (hunk 3 above)
Comment (av8ramit) [inline RELEASE.md]: Done.

File: RELEASE.md (hunk 4 above)
Comment (av8ramit) [inline RELEASE.md]: Done.

File: RELEASE.md (hunk 2 above)
Comment (av8ramit) [inline RELEASE.md]: Done.

Comment (caisq) [inline RELEASE.md]: oops, sorry for the grammar errors. please fix when you add them :) thanks

Comment (av8ramit) [inline RELEASE.md]: Done.

**Review by yifeif — APPROVED** [review]

**Review by caisq — APPROVED** [review]

**Review by xiejw — COMMENTED** [review]
LG for canned Estimator.

**Pull request merged**